### ISSUE TITLE
How to generate my "Trainval_GT_VCOCO.pkl"and "Trainval_Neg_VCOCO.pkl"

### ISSUE THREAD

**Issue by BestSongEver** [vt-vl-lab/iCAN#38]
Hi @gaochen315 
Sorry to bother. I have a quick question that maybe you can help.
I wonder how can i generate my own  "Trainval_GT_VCOCO.pkl"and "Trainval_Neg_VCOCO.pkl"?
To my understanding, **i can install Faster R-CNN and run it on test set of VCOCO** to get the test data  (to obtain "Test_Faster_RCNN_R-50-PFN_2x_VCOCO.pkl") . 
But how can i generate the training data?

Thanx a lot.
